src/components/stepper/index.js

```diff
@@ -4,7 +4,7 @@ import Button from "../button";
 import "./style.css";
 import { useState } from "react";
 
-const Stepper = ({ header, children, onComplete }) => {
+const Stepper = ({ data, header, children, onComplete }) => {
```

Comment:
data is very generic, can you be a bit more specific of what kind of data this stepper needs? maybe a comment above (i know it'd be solved with typescript)